cardano-cli/src/Cardano/CLI/Option/Flag.hs

```diff
@@ -65,4 +68,6 @@ mkFlag longName format value =
 
 isDefault :: Flag a -> Flag a
 isDefault ff =
-  ff & the @"options" . the @"isDefault" .~ True
+  ff
+    { Z.options = (Z.options ff){Z.isDefault = IsDefault}
```

Comment:
I much prefer lenses for nested record update because it more matches the intent of what we're doing, which is to update the `.options.isDefault` record to a specific value.

Without lenses we have to pack and unpack records manually and the intent is lot in the noise.